Frends.AzureBlobStorage.DeleteBlob/Frends.AzureBlobStorage.DeleteBlob/Definitions/Options.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.DeleteBlob
+{
+    /// <summary>
+    /// Options-class for DeleteBlob-task.
+    /// </summary>
+    public class Options
+    {
+        /// <summary>
+        /// Connection string to Azure storage.
+        /// </summary>
+        [PasswordPropertyText]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string ConnectionString { get; set; }

```

Comment:
This looks like something that must be in input, as without this it is impossible to run the task

Reply:
Moved to input.